echopop/extensions/inversion/patcher.py

```diff
@@ -0,0 +1,48 @@
+from pathlib import Path
+from typing import Any, Dict, Union
+
+import pandas as pd
+
+from ...survey import Survey
+
+
+def inversion_pipeline(
```

Comment:
The way things are designed now are _completely separate_ from the hake component, which is not what I envisioned. I think it is fine as an intermediate step, but ultimately a more modularized framework will be needed to make the package more useful as an open-source project. Let's talk more about this when we meet.